fluent-plugin-enhance-k8s-metadata/lib/sumologic/kubernetes/cache_strategy.rb

```diff
@@ -0,0 +1,35 @@
+module SumoLogic
+  module Kubernetes
+    # module for caching strategy
+    module CacheStrategy
+      require 'lru_redux'
+      require_relative 'reader.rb'
+
+      CACHE_TYPE_POD_LABELS = 'pod_labels'.freeze
+      CACHE_TYPE_OWNER_REFS = 'owner_refs'.freeze
+
+      def init_cache
+        @all_caches = {
+          CACHE_TYPE_POD_LABELS => LruRedux::TTL::ThreadSafeCache.new(@cache_size, @cache_ttl),
+          CACHE_TYPE_OWNER_REFS => LruRedux::TTL::ThreadSafeCache.new(@cache_size, @cache_ttl)
```

Comment:
What does owner refs do? Don't see any usage here?

Reply:
Planning to use it later for deployment lookup. I can remove it for now